estate/models/res_users.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    property_ids = fields.One2many("estate.property", "salesman", domain=['|', ('state', '=', 'New'), ('state', '=', 'Offer Received')])
```

Comment:
Quite complicate to read. You can use a "in" operator inside your domain.
```suggestion
    property_ids = fields.One2many("estate.property", "salesman", domain=['('state', 'in', ['New',  'Offer Received'])])
```

Reply:
By the way, you should only use lowercase letters for the values of the selection fields. They shouldn't be displayed for users and so do not have to be pretty.